### PR TITLE
Rate limit tail responses to one every 10ms

### PR DESCRIPTION
**Pull request by blinktag**

### CODE REVIEW

File: tail.go [hunk 1]
```diff
@@ -17,14 +17,26 @@ import (
 	"github.com/streamdal/snitch-go-client/validation"
 )
 
-const NumTailWorkers = 2
+const (
+	// NumTailWorkers is the number of tail workers to start for each tail request
+	// The workers are responsible for reading from the tail channel and streaming
+	// TailResponse messages to the snitch server
+	NumTailWorkers = 2
```
Comment (dselans) [inline tail.go:24]: Would there ever be a need to have more than 2? Just curious.

File: tail.go [hunk 2]
```diff
@@ -45,8 +57,22 @@ func (s *Snitch) sendTail(aud *protos.Audience, pipelineID string, originalData
...
+		tail.ShipResponse(tr)
+	}
+}
+
+func (t *Tail) ShipResponse(tr *protos.TailResponse) {
+	// If we're sending too fast, drop the message
+	if time.Since(t.lastMsg).Milliseconds() < MinTailResponseIntervalMS {
+		// TODO: we should notify the snitch server that we're dropping messages somehow
+		// TODO: but this needs to be done in a way that only sends once in a while
+		t.log.Debugf("Dropping tail response for %s, too fast", tr.PipelineId)
```
Comment (dselans) [inline tail.go:70]: Let's do this as a `Warnf` for now.

We can also expose a new metric for "num dropped tail messages". We can display that in the UI somewhere?

**Review by dselans — APPROVED** [review]
Nice! <3 

One minor comment (re: todo)

File: tail.go (hunk 2 above)
Comment (blinktag) [reply tail.go]: Good idea, done!

**Pull request merged**